ts/node.ts

```diff
@@ -1,44 +0,0 @@
-function getGlobal(): any | undefined {
```

Comment:
I don't believe this is needed anymore now that

1. Node has improved its `fetch` support a lot
2. We use GRPC web

Should be safe to remove.